openbb_terminal/miscellaneous/library/trail_map.csv

```diff
@@ -430,6 +433,19 @@ stocks.gov.toplobbying,openbb_terminal.stocks.government.quiverquant_model.get_t
 stocks.ins.act,openbb_terminal.stocks.insider.businessinsider_model.get_insider_activity,openbb_terminal.stocks.insider.businessinsider_view.insider_activity
 stocks.ins.lins,openbb_terminal.stocks.insider.finviz_model.get_last_insider_activity,openbb_terminal.stocks.insider.finviz_view.last_insider_activity
 stocks.ins.print_insider_data,openbb_terminal.stocks.insider.openinsider_model.get_print_insider_data,openbb_terminal.stocks.insider.openinsider_view.print_insider_data
+stocks.ins.stats,openbb_terminal.stocks.insider.sdk_helper.stats,
+stocks.ins.filter,openbb_terminal.stocks.insider.sdk_helper.insider_filter,
+stocks.ins.lcb,openbb_terminal.stocks.insider.sdk_helper.lcb,
```

Comment:
@jmaslek you decided to have multiple functions - being each function for a different type of insider - so we can leverage the docstring? Otherwise, we could call `get_print_insider_data` directly